Makefile

```diff
@@ -0,0 +1,242 @@
+#  and a popup will appear. Click 'Unlock' and then click 'Register' to
+#  register the instance as a mediator.
+#
+#  b) While still in the Account screen, press CMD+D and follow the same
+#  steps as above to register the instance as a refund agent.
+#
+# When the steps above are complete, your localnet should be up and
+# ready to use. You can now test in isolation all Bisq features and use
+# cases.
+#
+
+
+# Set up everything necessary for deploying your localnet. This is the
+# default target.
+setup: build localnet
+
+clean: clean-build clean-localnet
+
+clean-build:
+	./gradlew clean
+
+clean-localnet:
+	rm -rf localnet
+
+# Build all Bisq binaries and generate the shell scripts used to run
+# them in the targets below
+build:
+	./gradlew build
+
+# Unpack and customize a Bitcoin regtest node and Alice and Bob Bisq
+# nodes that have been preconfigured with a blockchain containing the
+# BSQ genesis transaction
+localnet:
+	# Unpack the old dao-setup.zip and move things around for more concise
+	# and intuitive naming. This is a temporary measure until we clean these
+	# resources up more thoroughly.
+	unzip docs/dao-setup.zip
+	mv dao-setup localnet
+	mv localnet/Bitcoin-regtest localnet/bitcoind
+	mv localnet/bisq-BTC_REGTEST_Alice_dao localnet/alice
+	mv localnet/bisq-BTC_REGTEST_Bob_dao localnet/bob
+	# Remove the preconfigured bitcoin.conf in favor of explicitly
+	# parameterizing the invocation of bitcoind in the target below
+	rm -v localnet/bitcoind/bitcoin.conf
+	# Avoid spurious 'runCommand' errors in the bitcoind log when nc
+	# fails to bind to one of the listed block notification ports
+	echo exit 0 >> localnet/bitcoind/blocknotify
+
+# Deploy a complete localnet by running all required Bitcoin and Bisq
+# nodes, each in their own named screen window. If you are not a screen
+# user, you'll need to run each of the make commands manually in a
+# separate terminal or as a background job.
+#
+# NOTE: You MUST already be attached to a screen session for the
+# following commands to work properly.
+deploy: setup
+	screen -t bitcoin make bitcoind
+	sleep 2    # wait for bitcoind rpc server to start
+	make block # generate a block to ensure Bisq nodes get dao-synced
+	screen -t seednode make seednode
+	screen -t seednode2 make seednode2
+	screen -t alice make alice
+	screen -t bob make bob
+	screen -t mediator make mediator
+
+bitcoind: localnet
+	bitcoind \
+		-regtest \
+		-prune=0 \
+		-txindex=1 \
+		-server \
+		-rpcuser=bisqdao \
+		-rpcpassword=bsq \
+		-datadir=localnet/bitcoind \
+		-blocknotify='localnet/bitcoind/blocknotify %s'
+
+seednode: build
+	./bisq-seednode \
+		--baseCurrencyNetwork=BTC_REGTEST \
+		--useLocalhostForP2P=true \
+		--useDevPrivilegeKeys=true \
+		--fullDaoNode=true \
+		--rpcUser=bisqdao \
+		--rpcPassword=bsq \
+		--rpcBlockNotificationPort=5120 \
+		--nodePort=2002 \
+		--userDataDir=localnet \
+		--appName=seednode
+
+seednode2: build
```

Comment:
Is the second seednode just for more testing of p2p forwarding? My setup has always been fine with just one so curious as to the extra benefit and if it is worth the resources.

Reply:
Only one seed node is ok.

Reply:
One seednode is okay but it leads to annoying error output in all the logs unless you remove one of [these lines](https://github.com/bisq-network/bisq/blob/8c8c7117d33c738bad86199a865d46bca7ddc9fd/core/src/main/resources/btc_regtest.seednodes#L16-L17)

Reply:
> One seednode is okay but it leads to annoying error output in all the logs

This is the only reason I spin it up. Gets everything closer to a no-broken-windows situation where the operator can assume there shouldn't be any (or many) errors / stack traces showing up in their Bitcoin and Bisq node logs.

Correct solution would perhaps be to do issuing a warning in the 2002 seednode log that the other well-known (3002) seednode cannot be found, and stopping trying after a reasonable several attempts instead of perpetually issuing error messages.

In the meantime, the `seednode2` target could be documented as optional and an explanatory note could be added about running to avoid the errors mentioned above.

The other reason I wanted a second seed node was to test what happens when one seednode is a dao fullnode and the other isn't. Seems to work fine, but the current configuration in the makefile spins them both up as fullnodes anyway.

Note that this is the same reason why `alice` is a dao fullnode but `bob` and `mediator` aren't: just to make sure we have this heterogenous setup in the out-of-the box localnet config. It better reflects the actual state of the production network, might help catch any issues.